gaia-sub-header.js

```diff
@@ -54,7 +54,7 @@ module.exports = component.register('gaia-sub-header', {
       font-weight: normal;
 
       color:
-        var(--color-zeta);
+        var(--color-epsilon);
```

Comment:
This should be changed to use the more abstract colors defined in gaia-theme. For example something like `--text-color` or `--text-color-minus`.